cmd/server/assets/codestatus/bulkissue.html

```diff
@@ -41,6 +45,19 @@
           This form will read the CSV contents and call the issue API one-by-one.
         </p>
         <form id="form">
+          <div class="form-label-group">
+            <input type="text" class="form-control" id="retry-code" placeholder="Retry code" required>
```

Comment:
From a security perspective, I think it'd be better for us to generate this and instruct the user to save it.

Reply:
We generate a value for the field in javascript below. I want to ensure the value does not come from the server. The field needs to be editable so the user could replace it with an already-save code?